src/sympc/protocol/falcon/falcon.py

```diff
@@ -318,7 +318,12 @@ def mul_malicious(
             )
             mask = parallel_execution(Falcon.falcon_mask, session.parties)(args)
 
-        eps_shares, delta_shares = zip(*mask)
+        # zip on pointers is compute intensive
+        eps_shares, delta_shares = [mask[0][0], mask[1][0], mask[2][0]], [
```

Comment:
Q: Can we split those into 2 lines? (also could you make a PR for SPDZ for this?)
```
eps_shares = [mask[0][0], mask[1], [0], mask[2][0]]
delta_shares = [mask[0][1], mask[1][1]. mask[2][1]]
```

Reply:
Sure George :+1: